opts/opts.go

```diff
@@ -203,6 +203,9 @@ func ValidatePath(val string) (string, error) {
 	}
 
 	splited := strings.SplitN(val, ":", 2)
+	if splited[0] == "" {
+		return val, fmt.Errorf("bad format for volumes: %s", val)
+	}
 	if len(splited) == 1 {
 		containerPath = splited[0]
 		val = path.Clean(splited[0])
```

Comment:
I'm pretty sure the logic in the 'else' part of this doesn't work when the input is `path:rw` because containerPath will be set to 'rw'.  The new testcases added doesn't test for this case, can we add it to see if I'm correct?